layers/+lang/ocaml/packages.el

```diff
@@ -32,6 +33,34 @@
       :modes merlin-mode
       :variables merlin-completion-with-doc t)))
 
+(defun ocaml/init-dune ()
```

Comment:
Please add code to load `dune-mode` after opening a dune file.
For an example please have a look at the package.el of `ietf` layer.

Reply:
Added.